ui/hooks/src/moderation-request.ts

```diff
@@ -255,7 +255,7 @@ export default {
             throw new Error('Unable to process your request right now. Please try again later');
           }
 
-          return modalName === 'report-modal' ? callback(true) : callback();
+          return modalName === 'report-modal' ? callback(true, dataToSign.reason) : callback();
         })
         .catch(error =>
           logger.error(`[${modalName}.tsx]: fetchRequest err %j`, error.message || ''),
```

Comment:
when you log the error don't lie about the source. this is not the `${modalName}.tsx` file. It's the moderation-request.modalClickHandler. also make sure that you pass only one parameter to the `.error` and it is a `string`

Reply:
gotcha.